eclair-node/src/main/scala/fr/acinq/eclair/api/serde/JsonSerializers.scala

```diff
@@ -68,6 +75,12 @@ class UInt64Serializer extends CustomSerializer[UInt64](_ => ( {
   case x: UInt64 => JInt(x.toBigInt)
 }))
 
+class BtcSerializer extends CustomSerializer[Btc](_ => ( {
```

Comment:
Definitely not for this PR, but I'm realizing only now that it would probably be much better if our serializers for bitcoin amounts returned an object that specified the unit, something like:
```json
{
  "amount": 0.3,
  "unit": "btc"
}
```
```json
{
  "amount": 0.3,
  "unit": "msat"
}
```

I'll create an issue to suggest that.